com.fdossena.flogo.yml

```diff
@@ -0,0 +1,45 @@
+app-id: com.fdossena.flogo
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node24
+command: run.sh
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
> I'm not sure about the sandbox permissions that I've used here. The application has to be able to read and save files, and registers the .flogo file extension (binary file with its own mime type). home would be enough, but I need my students to be able to read and save programs to flash drives, host seems to be the only option, unless there's special permissions for removable drives.

Does it not use filechoosers?

Reply:
Yes, but file associations don't seem to work without this, at least not outside of folders that I explicitly allow.

I don't like it but I'm not sure how else it could be handled.

Reply:
What do you mean by file associations don't work? What does that have to with this?

Reply:
If I remove `--filesystem=host`, when I double click a file in my file manager, the application receives the path of the file, but it then fails to read it because it can't access that path

Reply:
This is the current desktop file.
```
[Desktop Entry]
Name=Flogo
Comment=Create and run programs using flow charts
Exec=run.sh %U
Terminal=false
Type=Application
Icon=com.fdossena.flogo
MimeType=application/x-flogo
Categories=Education
StartupWMClass=Flogo
```

Reply:
Hold on, the one it installed in the system does have the --file-forwarding option, it runs this command
`/usr/bin/flatpak run --branch=master --arch=x86_64 --command=run.sh --file-forwarding com.fdossena.flogo @@u %U @@`

Reply:
sounds like a bug... maybe something is wrong with your portals or environment.

Can you remove it temporarily so that i can test a build?

Reply:
Done. I can also add some debugging prints in the app if you want